source/how-tos/boot-acrn.rst

```diff
@@ -0,0 +1,357 @@
+.. _boot-acrn:
+
+Boot ACRN Hypervisor
+------------------------
+
+Step-by-Step guide to build a custom |SPN| to boot ACRN and provision eMMC on |UP2|.
+
+Prerequisites
+^^^^^^^^^^^^^^^^^
+
+Before you start, prepare the following items:
+
+* ACRN images for |UP2| - URL TBD
+
+* Serial debug adaptor for UART0 on |UP2| - Required to trigger fastboot and perform firmware update
+
+* Micro USB cable (type A to microB)
+
+* USB key
+
+* **RECOMMENDED:** A SPI programmer in case you brick |UP2| and need to restore factory BIOS. See |UP2_FLASHING| for details.
+
+.. |UP2_FLASHING| raw:: html
+
+   <a href="https://wiki.up-community.org/BIOS_chip_flashing_on_UP_Squared" target="_blank">BIOS chip flashing</a>
+
+
+For more details on setting up the hardware, see :ref:`up2-board`.
+
+
+Build |SPN| 
+^^^^^^^^^^^^^^^^
+
+Build and stitch a custom |SPN| with fastboot support
+
+**Step 1:** Install tool chains and clone SBL source code from GitHub
+
+  Run::
+
+    sudo apt-get install -y build-essential iasl python uuid-dev nasm openssl gcc-multilib qemu

```

Comment:
This guide is for building slimboot to boot ACRN right? Can we direct the user to other documentation for this step instead of producing it here? My concern is that the instructions for this step may be out of date at some point and might not get updated with the other docs.